File: python/Kode Program/Electronic.py
# Class Electronic
# Kelas dasar untuk semua produk elektronik.
class Electronic:
    def __init__(self, id=0, name="", brand="", year=0):
        self.id = id
        self.name = name
        self.brand = brand
        self.year = year

    # Getter
    def get_id(self): return self.id
    def get_name(self): return self.name
    def get_brand(self): return self.brand
    def get_year(self): return self.year

    # Setter
    def set_name(self, n): self.name = n
    def set_brand(self, b): self.brand = b
    def set_year(self, y): self.year = y

File: php/Kode Program/Electronic.php
<?php
class Electronic {
    protected $id;
    protected $name;
    protected $brand;
    protected $year;

    public function __construct($id, $name, $brand, $year) {
        $this->id = $id;
        $this->name = $name;
        $this->brand = $brand;
        $this->year = $year;
    }

    // Getter
    public function getId() { return $this->id; }
    public function getName() { return $this->name; }
    public function getBrand() { return $this->brand; }
    public function getYear() { return $this->year; }
}
?>